src/App.js

```diff
@@ -8,7 +8,7 @@ import ContentDisplay from "./components/ContentDisplay";
 import MobileContentDisplay from "./components/MobileContentDisplay";
 
 function App() {
-  if (window.screen.width <= 1280) {
```

Comment:
good change! just read, and it seems like innerWidth is the general better width to use.